kolibri/plugins/assets/src/kolibri_module/kolibri_module.js

```diff
@@ -58,6 +58,15 @@ KolibriModule.prototype.stop_listening = function(event, method) {
     Kolibri.stop_listening(event, this, method);
 };
 
+/**
+ * Convenience method to fire an event on the global Mediator and pass in a data payload.
+ * @param {string} event - the event name
+ * @param [arguments] arguments - additional arguments to the event handler.
+ */
+KolibriModule.prototype.trigger = function() {
```

Comment:
This is a useful method and should be kept, @MCGallaspy - we may want to refactor it based on @indirectlylit's conversion of Backbone to Vue, for event handling, however.